### PR TITLE
branch major upgrade jobs for 4.17 CNO presubmits

### PR DESCRIPTION
**Pull request by maiqueb**

### CODE REVIEW

**Comment by maiqueb** [comment]
/cc @jluhrsen 

Could you help me out in this one ? I've noticed you've done [similar things in the past](https://github.com/openshift/release/pull/46748).

**Comment by jluhrsen** [comment]
@maiqueb , can you just make one commit. just do the changes in ci-operator/config/ files, then run `make jobs`. you shouldn't
need to do any manual changes to the ci-operator/jobs/ files

**Comment by maiqueb** [comment]
> @maiqueb , can you just make one commit. just do the changes in ci-operator/config/ files, then run `make jobs`. you shouldn't need to do any manual changes to the ci-operator/jobs/ files

I just did. No changes. I still squashed the commits together. 

There's something wrong, but I can't pinpoint it.

**Comment by jluhrsen** [comment]
> > @maiqueb , can you just make one commit. just do the changes in ci-operator/config/ files, then run `make jobs`. you shouldn't need to do any manual changes to the ci-operator/jobs/ files
> 
> I just did. No changes. I still squashed the commits together.
> 
> There's something wrong, but I can't pinpoint it.

where are you seeing something wrong?

**Comment by maiqueb** [comment]
> where are you seeing something wrong?

The rehearsals are failing, but I didn't have the time to understand why.

**Comment by jluhrsen** [comment]
> > where are you seeing something wrong?
> 
> The rehearsals are failing, but I didn't have the time to understand why.
> 
> I'll be on PTO for 2 weeks, so I won't be able to progress during this time.
> 
> Feel free to take over this PR if the team suddenly needs this.

they are failing for real reasons (e.g., the cluster upgrade failed). might have some trouble brewing
with 4.16->4.17 upgrades? but this PR seems right.

/lgtm
/approve

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master__4.17-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+base_images:
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
```
Comment (jcaamano) [inline ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master__4.17-upgrade-from-stable-4.16.yaml:5]: I think thid should be `tag: base-rhel9`

**Comment by jluhrsen** [comment]
@maiqueb , I didn't have auth to update this PR and since you were on PTO I just made [this new one](https://github.com/openshift/release/pull/53592) and it
go merged. You can close this one now. Thanks for doing the work though.

**Comment by maiqueb** [comment]
Abandoned in-lieu of https://github.com/openshift/release/pull/53592.

Thanks @jluhrsen / @jcaamano

**Pull request closed without merging**